ARM.CMSIS.pdsc

```diff
@@ -15,6 +15,7 @@
        - Support for int16 convolution and fully connected for reference implementation
        - Support for DSP extension optimization for int16 convolution and fully connected
        - Support for dilation for int8 convolution
+       - Support for dilation for int8 depthwise convolution
```

Comment:
Please rephrase it as 'Support dilation for int8 depthwise convolution'. The same for Convolution

Reply:
Done